src/hooks/usePocketbase.ts

```diff
@@ -0,0 +1,52 @@
+"use client";
+import PocketBase from "pocketbase";
+import { useState, useEffect } from "react";
+import { PB_URL } from "@/lib/pb";
+
+const usePocketbase = () => {
+  const pb = new PocketBase(PB_URL);
+  if (typeof window !== "undefined") {
+    // BUG why do we need this conditional with use client?
```

Comment:
From what I understand, client components still render on the server. The difference is that its javascript is downloaded to the client and doesn't hit the server again on soft navigations.
https://nextjs.org/docs/app/building-your-application/rendering/client-components

The only way to prevent this is to dynamically import this hook - which I don't think you can do 🤔

Reply:
That makes a lot more sense. I got a bit confused with the error message here